src/command/expire/expire.c

```diff
@@ -863,6 +863,88 @@ removeExpiredBackup(InfoBackup *infoBackup, const String *adhocBackupLabel, unsi
     FUNCTION_LOG_RETURN_VOID();
 }
 
+
+/***********************************************************************************************************************************
+Remove expired backup history manifests from repo
+***********************************************************************************************************************************/
+static void
+removeExpiredHistory(InfoBackup *infoBackup, unsigned int repoIdx)
+{
+    FUNCTION_LOG_BEGIN(logLevelDebug);
+        FUNCTION_LOG_PARAM(INFO_BACKUP, infoBackup);
+        FUNCTION_LOG_PARAM(UINT, repoIdx);
+    FUNCTION_LOG_END();
+
+    ASSERT(infoBackup != NULL);
+
+    MEM_CONTEXT_TEMP_BEGIN()
+    {
+        if (cfgOptionIdxTest(cfgOptRepoRetentionHistory, repoIdx))
+        {
+            // Get all the current backups in backup.info - these will not be expired.
```

Comment:
Remove periods from single sentence comments -- check all comments below.

Reply:
Done in 001035e.